src/NServiceBus.Core/TaskEx.cs

```diff
@@ -1,11 +1,14 @@
 namespace NServiceBus
 {
+    using System;
     using System.Threading.Tasks;
 
     static class TaskEx
     {
+        const string TaskIsNullExceptionMessage = "You must return a Task or mark the method as async.";
```

Comment:
Not any ruder than the current message. I will pull out the "You must". We are just stating a requirement.